pyproject.toml

```diff
@@ -142,3 +191,111 @@ reportUnusedParameter = false
 reportAny = false
 reportImplicitOverride = false
 reportAttributeAccessIssue = false  # adding our own private property on click commands
+
+# New Hatch environment configurations below
+
+[[tool.hatch.envs.test.matrix]]
+python = ["3.11", "3.12"]
+
```

Comment:
Using a matrix here might cause issues with the ci job, which I think only has one version of python. I could be wrong. If I am right, we could just assume the current version of python if the right one and skip the matrix.

Reply:
Going to remove this as this not something we need.